xml/article_sap_trento.xml

```diff
@@ -928,13 +929,26 @@ Warning: Permanently added '<replaceable>TRENTO_AGENT_IP</replaceable>' (ECDSA)
         <para> Ensure the &t.server; is running. </para>
       </step>
       <step>
-        <para> On the &t.server; host start the update: </para>
-        <screen>&trentoagentprompt;sudo curl -sfL https://raw.githubusercontent.com/trento-project/trento/main/install-server.sh | bash
+        <para>Export the <envar>KUBECONFIG</envar> environment variable for the
+          same user that installed K3s:
+        </para>
+        <screen>export KUBECONFIG=/etc/rancher/k3s/k3s.yaml</screen>
+      </step>
+      <step>
+        <para>
+          With the same user that installed K3s, start the update
+          n the &t.server; host:
+        </para>
+        <screen>HELM_EXPERIMENTAL_OCI=1 helm upgrade \
+   --install <replaceable>TRENTO_SERVER_HOSTNAME</replaceable> \
+   oci://registry.suse.com/trento/trento-server \
+   --version 0.3.5 \
+   --set-file trento-runner.privateKey=<replaceable>PRIVATE_SSH_KEY</replaceable>
         </screen>
       </step>
       <step>
         <para>Check the &t.server; processes:</para>
-        <screen>&trentoagentprompt;sudo kubectl get pod</screen>
+        <screen>&trentoagent.prompt;sudo kubectl get pod</screen>
       </step>
```

Comment:
Why is Root needed here? That should not be needed, the helm upgrade and the kubectl are run as same user without root needed.

Remove sudo

Reply:
Remember, root != trentoadmin. :slightly_smiling_face: I can remove the prompt and the sudo.